src/verinfast/config.py

```diff
@@ -398,7 +400,7 @@ def handle_args(self, args):
                 self.truncate_findings = False
 
     def is_path_remote(self) -> bool:
```

Comment:
we should rename the function too to avoid confusion.

Reply:
fixed